packages/bpk-component-infinite-scroll/README.md

```diff
@@ -160,12 +162,13 @@ Updates the internal array and triggers all listeners.
 
 ## Props
 
-| Property               | PropType                | Required | Default Value |
-| ---------------------- | ----------------------- | -------- | ------------- |
-| dataSource             | instanceOf(DataSource)  | true     | -             |
-| elementsPerScroll      | number                  | false    | 5             |
-| onScroll               | func                    | false    | null          |
-| onScrollFinished       | func                    | false    | null          |
-| renderLoadingComponent | func                    | false    | null          |
-| renderSeeMoreComponent | func                    | false    | null          |
-| seeMoreAfter           | number                  | false    | null          |
+| Property                | PropType               | Required | Default Value |
+| ----------------------- | ---------------------- | -------- | ------------- |
+| dataSource              | instanceOf(DataSource) | true     | -             |
+| initiallyLoadedElements | number                 | false    | 5             |
```

Comment:
Just one small thing here. We put required props in alpabethical order first and then non required props in alpabethical order, so `initiallyLoadedElements` should come after `elementsPerScroll`